doc/release_notes/release_notes_2.2.rst

```diff
@@ -0,0 +1,118 @@
+.. _release_notes_2.2:
+
+ACRN v2.1 (Sep 2020)
+####################
+
+We are pleased to announce the release of ACRN version 2.2
+
+
+ACRN is a flexible, lightweight reference hypervisor, built with real-time and safety-criticality in mind, optimized to streamline embedded development through an open source platform. Check out the What is ACRN for more information. All project ACRN source code is maintained in the https://github.com/projectacrn/acrn-hypervisor repository and includes folders for the ACRN hypervisor, the ACRN device model, tools, and documentation. You can either download this source code as a zip or tar.gz file (see the ACRN v2.2 GitHub release page) or use Git clone and checkout commands:
+
+git clone https://github.com/projectacrn/acrn-hypervisor
+cd acrn-hypervisor
+git checkout v2.2
```

Comment:
This is different than the instructions we give in our GSG. Let's settle on a single one.